ctagsd/tests/main.cpp

```diff
@@ -229,7 +229,7 @@ TEST_FUNC(TestLSPLocation)
     if(resolved) {
         completer->get_completions(resolved, wxEmptyString, wxEmptyString, candidates, { "LSP" });
     }
-    CHECK_BOOL(resolved);
+    CHECK_NOT_NULL(resolved);
```

Comment:
More appropriate meaning, and `operator bool` is `explicit`.
(Apply to whole file).